facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -46,9 +46,9 @@ <h4 class="trail-text">@Html(trailText)</h4>
     }
 }
 
-@headlineAndTrailWithCutout(card: ContentCard) = {
+@headlineAndTrailWithCutout(card: ContentCard, withImage: Boolean) = {
     @fullRow(Seq("facia-card__text")) {
-        @headline(card)
+        @headline(card, withImage)
```

Comment:
Could be slightly clearer to use a named param here, i.e. `@headline(card, largeHeadline = withImage)`